scripts/z2

```diff
@@ -6,5 +6,5 @@ export ZQ2_BASE
 
 if [[ -d $ZQ2_BASE ]]; then
   cd $ZQ2_BASE/zq2
-  cargo run -q --bin z2 -- $*
+  cargo run --release -q --bin z2 -- $*
```

Comment:
This is fixed. I made the release profile optional for the z2 converter and document how to use it. The reason why I used the release build is because optimized binaries were ~x10 faster in converting the zq1 to z2 mainnet persistence.